crates/subspace-node/src/chain_spec.rs

```diff
@@ -179,7 +180,14 @@ pub fn gemini_3h_compiled() -> Result<GenericChainSpec<RuntimeGenesisConfig>, St
                     enable_dynamic_cost_of_storage: false,
                     enable_balance_transfers: true,
                     enable_non_root_calls: false,
-                    confirmation_depth_k: 100, // TODO: Proper value here
+                    // TODO: Proper value here
+                    confirmation_depth_k: 100,
+                    // TODO: Proper value here
+                    rewards_config: RewardsConfig {
+                        remaining_issuance: 1_000_000_000 * SSC,
```

Comment:
Do vesting allocations subtract from this value too? They should. For the next block after genesis `remaining_issuance` should already be much lower.

Reply:
Vesting is initialized at genesis and is never part of `remaining_issuance` in the first place. These values are placeholders for now and will need to be set to proper values before launch.